projections/mit/SUPPORT.md

```diff
@@ -0,0 +1,25 @@
+# TODO: The maintainer of this repo has not yet edited this file
+
+**REPO OWNER**: Do you want Customer Service & Support (CSS) support for this product/project?
+
+- **No CSS support:** Fill out this template with information about how to file issues and get help.
+- **Yes CSS support:** Fill out an intake form at [aka.ms/spot](https://aka.ms/spot). CSS will work with/help you to determine next steps. More details also available at [aka.ms/onboardsupport](https://aka.ms/onboardsupport).
+- **Not sure?** Fill out a SPOT intake as though the answer were "Yes". CSS will help you decide.
+
+*Then remove this first heading from this SUPPORT.MD file before publishing your repo.*
+
+# Support
+
+## How to file issues and get help  
+
+This project uses GitHub Issues to track bugs and feature requests. Please search the existing 
+issues before filing new issues to avoid duplicates.  For new issues, file your bug or 
+feature request as a new Issue.
+
+For help and questions about using this project, please **REPO MAINTAINER: INSERT INSTRUCTIONS HERE 
+FOR HOW TO ENGAGE REPO OWNERS OR COMMUNITY FOR HELP. COULD BE A STACK OVERFLOW TAG OR OTHER
+CHANNEL. WHERE WILL YOU HELP PEOPLE?].
+
+## Microsoft Support Policy  
+
+Support for this **PROJECT or PROJECT** is limited to the resources listed above.

```

Comment:
What's this mean "**PROJECT or PROJECT**"?

Reply:
I think the guidance from @jhornick-ms is that they want the maintainer to clarify if it's a product or a project... products might be something like Cosmos DB vs a project like "some Cosmos integration for my product", maybe?

I'll let @jhornick-ms comment

Reply:
I wondered, in which case this should probably be "PROJECT or PRODUCT" (unless my eyes are deceiving me, it says "project" twice).

Reply:
@andschwa - @jeffwilcox  is right - this verbiage was intended to allow the repo owners to indicate whether this is a project, or something that rises to the level of "product" which is more likely - but not required - to carry official support.  There are entities which differentiate between the two, so we started with this in an attempt to embrace those terms.  If it's confusing, we could alter to "Support for the contents of this repo is limited to the resources listed above."

Reply:
It's a typo.